app/core/Encryptor/constants.ts

```diff
@@ -17,10 +17,34 @@ export enum KeyDerivationIteration {
   OWASP2023Default = 900_000,
 }
 
+/**
+ * Supported SHA algorithms in react-native-aes v3.0.3
+ */
+export enum ShaAlgorithm {
+  Sha256 = 'sha256',
+  Sha512 = 'sha512',
+}
+
+/**
+ * Supported cipher algorithms in react-native-aes v3.0.3
+ *
+ * Important Note: Make sure to validate the compatibility of the algorithm with the underlying library.
+ * The react-native-aes-crypto does a string validation for the algorithm, so it's important to make sure
+ * we're using the correct string.
+ *
+ * References:
+ * - encrypt: https://github.com/MetaMask/metamask-mobile/pull/9947#:~:text=When-,encrypting,-and%20decypting%20the
+ * - decrypt: https://github.com/MetaMask/metamask-mobile/pull/9947#:~:text=When%20encrypting%20and-,decypting,-the%20library%20uses
+ */
+export enum CipherAlgorithm {
+  cbc = 'aes-cbc-pkcs7padding',
+  ctr = 'aes-ctr-pkcs5padding',
```

Comment:
If you want to include this:

```suggestion
  Cbc = 'aes-cbc-pkcs7padding',
  Ctr = 'aes-ctr-pkcs5padding',
```

Reply:
Will be resolved in a follow up PR